org.eclipse.winery.repository.ui/src/app/wineryRepository.component.ts

```diff
@@ -20,13 +21,15 @@ import { backendBaseURL } from './configuration';
         ExistService
     ]
 })
+
 /*
  * This component represents the root component for the Winery Repository.
  */
 export class WineryRepositoryComponent implements OnInit {
     // region variables
     name = 'Winery Repository';
-    isBackendAvailable = false;
+    isBackendAvailable = BackendStates.Undefined;
```

Comment:
could you rename the variable `isBackendAvaiable` to `backendState`? This would increase consistency to your new Enum.

Reply:
Or `backendAvailabilityState`, because we need the `backendStates` for reference in the HTML-Template. If we use `backendState` and `backendStates`, it is very confusing.